src/AasxServerDB/EditDB.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AasCore.Aas3_0;
+using AdminShellNS;
+using Microsoft.EntityFrameworkCore;
+
+namespace AasxServerDB
+{
+    public class EditDB

```

Comment:
The naming is a bit odd. You created a class called **Edit** and it has a method **Edit**. That is a duplication. Can you adjust the naming?
Just as a quick idea:
- class `AASXDatabaseManager`
- method `Update`